content/post/092-launching-testground.md

```diff
@@ -0,0 +1,118 @@
+
+Prior to Testground, distributed testing used to imply deploying fleets of daemons. You’d expose every single internal component, method and configuration parameter over an API, and command-and-control them via an external script acting like a puppeteer. But in Testground, **you write test plans just as you’d write unit tests.**
+
+We hear that the Testground approach feels strikingly natural to developers. Why? Because **test plans call your internal APIs directly,** and they leverage **a distributed synchronisation API to coordinate** with other instances participating in the test run. Thus, test cases turn into distributed state machines that follow a precise choreography.
+
+At present, [**you can write test plans in Go**](https://github.com/testground/sdk-go). But implementing a Testground SDK for another language/runtime is dead simple! In fact, the community is hacking away at a TypeScript SDK for node and the browser.
+
+Because real decentralised systems are at the mercy of network conditions, **Testground can shape network traffic** by flipping connectedness, changing IP addresses at runtime, or setting latency, jitter, bandwidth, etc.
+
+And since not all test scenarios require thousands of instances, you can **run test plans locally as executables or Docker containers,** saving the overhead, time and cost of resorting to cluster deployments for when you need to scale beyond ~300 instances, or so. For those larger cases, **Testground supports Kubernetes out-of-the-box.** 
+
+To illustrate the above, consider a 200-instance test run. With Kubernetes, it could take 1-2 minutes to ship a Docker image to a remote registry, schedule a Kubernetes job, and wait for the workload to start. With the local Docker runner, that overhead is zero, and the run is immediate. It doesn’t sound like much, but developers know the death by a thousand cuts (or context switches) such delays do to one’s productivity and workflow.
+
+The cherry on the top of all this is the **observation pipeline (experimental)**. Test plans can record raw data points or aggregate metrics (histograms, [EWMA](https://en.wikipedia.org/wiki/EWMA_chart), counters, etc.) about the system under test, as well as emit arbitrary output assets. **The Testground platform will harvest them and make them available for one-stop collection.**
+
+![Testground Architecture](/092-launching-testground/launch_testground_architecture.png)
+
+<p style="text-align:center;font-size:75%;"> Architecture diagram, focused on a test plan’s inputs and outputs, highlighting the circuitry of the observability pipeline</p>
+
+## Getting started
+
+Want to hit the ground running? Try the following:
+
+1. Head over to the [Getting started](https://docs.testground.ai/getting-started) section of our docs.
+2. Make sure to check out the [testground/testground](https://github.com/testground/testground)  repo on GitHub.
```

Comment:
can we add another bullet:

1. Review the [concepts and architecture](https://docs.testground.ai/concepts-and-architecture) behind Testground.